service/k8sclient/kubectl/kubectl.go

```diff
@@ -258,6 +259,10 @@ func run(ctx context.Context, kubectlCmd []string, args []string, stdin interfac
 	cmd.Stdin = &inBuf
 	cmd.Stdout = &outBuf
 	cmd.Stderr = &errBuf
+	cmd.Env = []string{
```

Comment:
⚠️ Sorry, that's incorrect. That completely replaces the environment, and we should add to it: get `os.Environ()`, iterate over it, find and update PATH, add to `cmd.Env`